packages/react-hooks/src/useMergedRefs.ts

```diff
@@ -1,13 +1,15 @@
 import { useCallback, Ref, MutableRefObject } from 'react';
+
 /**
  * React hook to merge multiple React refs (either MutableRefObjects or ref callbacks) into a single ref callback that
  * updates all provided refs
- * @param refs- Refs to collectively update with one ref value.
+ * @param refA- A first ref to collectively update with one ref value.
+ * @param refB- A second ref to collectively update with one ref value.
  */
-export function useMergedRefs<T>(...refs: Ref<T>[]): (instance: T) => void {
+export function useMergedRefs<T>(refA: Ref<T>, refB: Ref<T>): (instance: T) => void {
   return useCallback(
     (value: T) => {
-      for (const ref of refs) {
+      for (const ref of [refA, refB]) {
```

Comment:
If there will be an agreement with change for me it makes sense to eliminate this loop

Reply:
The change LGTM